sympy/sets/sets.py

```diff
@@ -746,6 +753,11 @@ def __iter__(self):
         else:
             raise TypeError("Not all constituent sets are iterable")
 
+    @property
+    def is_empty(self):
+        if all(s.is_empty == False for s in self.sets):
+            return False
```

Comment:
This should be any rather than all. Also maybe it makes sense to use `fuzzy_or`.

Reply:
Your comment had me scratching my head until I realized that I was constantly thinking about non-emptyness, while your comment is about emptyness. Since the property is `is_empty` it's indeed cleaner and simpler to `return fuzzy_or(s.is_empty ...)`.
I'll change it, thanks.